README.md

```diff
@@ -95,6 +95,18 @@ analysis-runner \
 For more examples (including for running an R script and dataproc), see the
 [examples](examples) directory.
 
+### GitHub Authorisation
+
+If you are submitting an analysis-runner job that needs to clone a private repository owned by populationgenomics on GitHub (eg submitting a script to analysis-runner from a private repository), please make sure that your configuration file contains the following section:
+
+```toml
+[infrastructure]
+git_credentials_secret_name = '<ask_software_team_for_secret_name>'
+git_credentials_secret_project = '<ask_software_team_for_secret_project>'
+```
+
+If you are specifying multiple configuration files, please make sure that this section appears in the final right-most config to avoid these settings being overwritten.
```

Comment:
Configs are merged by being folded right to left, so if you set `--config=A,B,C` then the values in C will overwrite those in B and A for any duplicate keys. I believe this is by design, and I've seen it documented somewhere in `cpg-utils`

Reply:
Right, but… What I don't understand is if you have another config file also specifying `infrastructure.git_credentials_secret_project = <something_else>` then one of them isn't going to work. So presumably you'll still have a problem.

Given that this is recent mostly unused functionality, I would have been surprised if other config files people were using also contained `git_credentials_secret_project` entries. However this would be the right workaround or at least alerts users to the problem, and I guess this is what @cassimons ran into, so :+1: I guess it's the right pointer to put in the documentation :shrug: